src/main/resources/application.yml

```
@@ -0,0 +1,2 @@
+spring.application.name=discodeit
+
```

properties에서 yml로 변경했다면 내용도 그에 맞게 들여쓰기형식으로 바꿔야할 것 같아요.  
```suggestion
spring:
  application:
    name: discodeit

```